res/functional-tests/configuration.toml

```diff
@@ -88,3 +88,21 @@ AccessTokenFile = '/tmp/edgex/secrets/AppService-functional-tests/consul-token'
 [Trigger]
```

Comment:
need to remove `ConfigAccessTokenFile ` and `AccessTokenFile ` from the above config.